spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -174,6 +174,45 @@
         expect(page).to have_current_path(idv_phone_url)
       end
     end
+
+    it 'shows a failure message after the maximum amount of retries', allow_browser_log: true do
```

Comment:
Do we already have coverage for this?

https://github.com/18F/identity-idp/blob/58b0d61d7b242ded697a70b35a3d4002e2f73ac4/spec/features/idv/doc_auth/document_capture_spec.rb#L91-L101

Reply:
That's a good point! That test is in the `desktop` context, while the new one is in the `mobile`. With that in mind, do you still think this test is redundant? I am open to the idea of deleting it, and curious to hear your thoughts.

Reply:
I think since the rate-limiting would apply regardless of device, it should suffice to have a single test.

Reply:
Thanks for confirming! I've just pushed up a commit deleting the test.